Changelog.rst

```diff
@@ -23,6 +25,8 @@ version NEXTVERSION
 * New class `cf.NetCDF4Array`
 * New class `cf.CFAH5netcdfArray`
 * New class `cf.CFANetCDF4Array`
+* Replace core `dask` functionality with that imported from `cfdm`
+  (https://github.com/NCAS-CMS/cf-python/pull/836)
```

Comment:
```suggestion
```

The new Changelog entry is now a duplicate - you'd already added one. So we should ditch one of them. For the record I prefer this message but the link to the Issue as per above, but removing either is fine.